internal/pkg/mapper/orchestrator/local_load.go

```diff
@@ -225,13 +225,13 @@ func (l *localLoader) getTargetConfig(configPath string) (*model.Config, error)
 
 func (l *localLoader) phasesDirs() []string {
 	// Check if blocks dir exists
-	if !l.Fs().IsDir(l.phasesDir) {
+	if !l.ObjectsRoot().IsDir(l.phasesDir) {
```

Comment:
Project uklada subory do `/`, a template do `/src`.
Tak som tuto metodu `Fs` premenoval na `ObjectsRoot`, aby to bolo jasnejsie.